_functions.sh

```diff
@@ -252,6 +259,16 @@ initialize_product_settings() {
       configurable_env_var "DEPLOYMENT_ES_EMBEDDED" true
       configurable_env_var "DEPLOYMENT_ES_IMAGE" "exoplatform/elasticsearch"
 
+      
+      env_var "DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED" false
+
+      if [[ ",$DEPLOYMENT_ADDONS:" =~ "exo-onlyoffice" ]]; then
```

Comment:
Why are `,` and `:` needed in the if expression ?

Reply:
they aren't, removed